pkg/cmd/roachtest/tests/cluster_to_cluster.go

```diff
@@ -588,8 +588,9 @@ func (rd *replicationDriver) setupC2C(
 		rd.t.L().Printf("Prom has started")
 	}
 	return func() {
-		rd.fetchDebugZip(ctx, rd.setup.src.nodes, "source_debug.zip")
-		rd.fetchDebugZip(ctx, rd.setup.dst.nodes, "dest_debug.zip")
+		debugCtx := context.Background()
```

Comment:
Can we set a reasonable timeout on this operation, just to be on the safe side?

Reply:
Done.